specification/manufacturingdatasolution/mds-service/data-plane/Microsoft.Industries.ManufacturingDataSolution/stable/2025-03-01/openapi.json

```diff
@@ -0,0 +1,3593 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.Industry.Manufacturing.MDS.Solution",
+    "version": "2025-03-01",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "https://mds-{mdsUniqueId}.{location}.cloudapp.azure.com/",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "name": "mdsUniqueId",
+        "in": "path",
+        "required": true,
+        "type": "string"
+      },
+      {
+        "name": "location",
+        "in": "path",
+        "required": true,
+        "type": "string",
+        "default": "mds.azure.com"
+      }
+    ]
+  },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "AadToken": [
+        "https://mds.azure.com/.default"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "AadToken": {
+      "type": "oauth2",
+      "description": "The Azure Active Directory OAuth2 Flow",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/v2.0/authorize",
+      "scopes": {
+        "https://mds.azure.com/.default": ""
+      },
+      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/token"
+    }
+  },
+  "tags": [],
+  "paths": {
+    "/admin/cleanup/{entityName}": {
+      "delete": {
+        "operationId": "AdminApis_DeleteOrCleanupEntityRequest",
+        "description": "Long-running resource delete operation template.",
```

Comment:
Please provide a proper description on this operation. And there are a couple more operations with "operation template" in the description -- these too should be fixed.